src/main/java/roomescape/RoomescapeController.java

```diff
@@ -0,0 +1,64 @@
+package roomescape;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@Controller
+public class RoomescapeController {
+    private List<Reservation> reservations = new ArrayList<>();
+
+    AtomicInteger atomic = new AtomicInteger(0);
+
+    @GetMapping("/")
+    public String home() {
+        return "home";
+    }
+
+    @GetMapping("/reservation")
+    public String reservation() {
+        return "reservation";
+    }
+
+    @GetMapping("/reservations")
+    @ResponseBody
+    public ResponseEntity<List<Reservation>> read() {
+        return ResponseEntity.ok(reservations);
+    }
+
+    @PostMapping("/reservations")
+    @ResponseBody
+    public ResponseEntity<Reservation> createReservation(@RequestBody Reservation reservation) {
+        if(reservation.getName().isEmpty() || reservation.getDate().isEmpty() || reservation.getDate().isEmpty()){
+            return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+        }
+
+        Reservation newReservation = new Reservation(atomic.incrementAndGet(), reservation.getName(), reservation.getDate(), reservation.getTime());
+        reservations.add(newReservation);
+
+        return ResponseEntity.created(URI.create("/reservations/" + newReservation.getId())).body(newReservation);
+    }
+
+    @DeleteMapping("/reservations/{id}")
+    @ResponseStatus (HttpStatus.NO_CONTENT)
```

Comment:
이렇게 응답 코드 반환을 할 수 있는 방법도 있었네요!
```suggestion
    @ResponseStatus(HttpStatus.NO_CONTENT)
```
형식을 맞추어 코드를 작성한다면 조금 더 예쁜 코드가 될 수 있을 것 같네요💫

Reply:
매의 눈..! 수정완료했습니다.
e882f4cacf3cf08d32589e291e5433dfd97902ab

Reply:
혹시 백앤드에서 주로 사용하시는 코드 포매터가 있나요?

Reply:
현재는 코드 포맷팅 규칙을 IntelliJ에서 적용하여 사용하고 있답니다